recipe/meta.yaml

```diff
@@ -19,14 +19,14 @@ requirements:
         - automake
         - libtool
         - pkg-config
-        - icu 56.*
+        - icu 56.* 56.*
```

Comment:
Ping @jakirkham - just a heads up - the pinning script can only do a string replacement in one way for one package, so if run and build requirements differ, we end up having this kind of issue.